src/scripts/main.js

```diff
@@ -1 +1,24 @@
 'use strict';
+
+const largeImg = document.getElementById('largeImg');
+const thumbs = document.getElementById('thumbs');
+
+function updateMainImage(src) {
+  largeImg.src = src;
+}
+
+thumbs.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  let target = e.target;
+
+  if (target.tagName === 'IMG') {
+    target = target.parentElement;
```

Comment:
The logic here assumes that if the target is an 'IMG', its parent is an 'A' tag. This is correct based on the task description, but ensure that the HTML structure matches this assumption.